homeassistant/components/sense/config_flow.py

```diff
@@ -34,13 +34,12 @@ class SenseConfigFlow(ConfigFlow, domain=DOMAIN):
 
     VERSION = 1
 
-    def __init__(self):
+    def __init__(self) -> None:
         """Init Config ."""
-        self._gateway = None
-        self._auth_data = {}
-        super().__init__()
+        self._gateway: ASyncSenseable | None = None
+        self._auth_data: dict[str, Any] = {}
```

Comment:
not sure the conventions for homeassistant but https://docs.python.org/3/library/typing.html#typing.TypedDict could make a more helpful type

Reply:
We don't use them much in the config flow, since the values are rather just written once and read once. I think extra typing is always nice, but since everything is in one file, I don't think it's it has that much benefit